### PR TITLE
Add pvc example

### PR DESCRIPTION
**Pull request by taozha2**

### CODE REVIEW

**Review by jiyang1011 — APPROVED** [review]

**Review by tdeng5 — APPROVED** [review]

**Review by Jiaxingla — APPROVED** [review]

File: include/cute/util/sycl_vec.hpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#else 
+template<class T, int N> using vector_t = sycl::vec<T,N>;
+#endif
+
+using float8 = vector_t<float, 8>;
+using short8 = vector_t<short, 8>;
+using ushort8 = vector_t<ushort, 8>;
+using int2_ = vector_t<int, 2>; //conflicts with vector_types
+using int8 = vector_t<int, 8>;
+using uint8 = vector_t<uint, 8>;
```
Comment (mehdi-goli) [inline include/cute/util/sycl_vec.hpp:17]:
```suggestion
using uint8 = vector_t<uint, 8>;

```

File: include/cute/atom/mma_atom.hpp [hunk 2]
```diff
@@ -946,4 +946,5 @@ print_latex_mma(LayoutC const& C, ThrIDC const& TC,  // (m,n) -> (tid,vid)  and
 #include <cute/atom/mma_traits_sm80.hpp>
 #include <cute/atom/mma_traits_sm90.hpp>
 #include <cute/atom/mma_traits_sm90_gmma.hpp>
+#include <cute/atom/mma_traits_xe.hpp>
```
Comment (mehdi-goli) [inline include/cute/atom/mma_atom.hpp:949]:
```suggestion
#if defined(CUTLASS_ENABLE_SYCL)
#include <cute/atom/mma_traits_xe.hpp>
#endif
```

File: include/cute/arch/mma_xe.hpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  CUTE_HOST_DEVICE static void
+  fma(float      & d,
+      short const& a,
+      int8  const& b,
+      float const& c)
+  {
+    d = intel_sub_group_bf16_bf16_matrix_mad_k16(a, b, c);
+  }
+};
+} //namespace cute
```
Comment (mehdi-goli) [inline include/cute/arch/mma_xe.hpp:56]:
```suggestion
} //namespace cute

```

File: include/cute/atom/copy_traits_xe.hpp [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+    template <class GEngine, class GLayout>
+    auto make_xe_2d_copy(Tensor<GEngine, GLayout> gtensor)
+    {
+        using GTensor = Tensor<GEngine, GLayout>;
+        using Traits = Copy_Traits<XE_2D_LOAD, GTensor>;
+        Traits traits{gtensor};
+        return Copy_Atom<Traits, typename GEngine::value_type>{traits};
+    }
+}
```
Comment (mehdi-goli) [inline include/cute/atom/copy_traits_xe.hpp:63]:
```suggestion
}

```

**Comment by rolandschulz** [comment]
What's the point of the intel_gpu_backend_enabling branch? It is just behind sycl-develop. Why would we want two different branches? Note that https://github.com/codeplaysoftware/cutlass-fork/pull/24 was already open. Containing the same code and trying to merge it into sycl-develop. Also we should try to avoid reviewing code which isn't actual part of this PR. All Mehdi's comments are for code part of https://github.com/codeplaysoftware/cutlass-fork/pull/23 and already addressed there. If we create PRs depending on other PRs we should put that in the description so that people know to only review the last commit(s).

**Comment by tdeng5** [comment]
> What's the point of the intel_gpu_backend_enabling branch? It is just behind sycl-develop. Why would we want two different branches? Note that #24 was already open. Containing the same code and trying to merge it into sycl-develop. Also we should try to avoid reviewing code which isn't actual part of this PR. All Mehdi's comments are for code part of #23 and already addressed there. If we create PRs depending on other PRs we should put that in the description so that people know to only review the last commit(s).

we use this branch for intel gpu backend related experiments and avoid be interrupted/interrupting sycl-develop branch, will merge our features back to master when the feature is mature enough.

**Review by rolandschulz — APPROVED** [review]

**Pull request merged**